charts/rancher-logging/106.0.0-rc.1+up4.10.0/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+annotations:
+  catalog.cattle.io/auto-install: rancher-logging-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/deploys-on-os: windows
+  catalog.cattle.io/display-name: Logging
+  catalog.cattle.io/kube-version: '>= 1.28.0-0 < 1.32.0-0'
+  catalog.cattle.io/namespace: cattle-logging-system
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/provides-gvr: logging.banzaicloud.io.clusterflow/v1beta1
+  catalog.cattle.io/rancher-version: '>= 2.10.0-0 < 2.11.0-0'
```

Comment:
Oh yeah - for 2.11 this forward port needs an additional change to patches to change this. Instead of this it should be:

```
>= 2.10.0-0 < 2.12.0-0
```

For now we can skip bumping the minimum side - that lets us test on 2.10 still. I think main shows as 2.11 now but just in case it's 2.10 this will allow it to work either way. Then closer to release - since we'll have more RCs likely anyway - we can bump the minimum to 2.11.